PowerPointLabs/PowerPointLabs/Views/AboutDialogBox.xaml

```diff
@@ -0,0 +1,78 @@
+                      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                      xmlns:controls="clr-namespace:MahApps.Metro.Controls;assembly=MahApps.Metro"
+                      mc:Ignorable="d"
+                      Height="205"
+                      Width="405"
+                      BorderThickness="1"
+                      BorderBrush="#0070c0"
+                      ResizeMode="NoResize"
+                      Title="About PowerPointLabs"
+                      WindowStartupLocation="CenterScreen"
+                      WindowTransitionsEnabled="False">
+    <controls:MetroWindow.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Colors.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/blue.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/baselight.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </controls:MetroWindow.Resources>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="29*"/>
+            <RowDefinition Height="12*"/>
+        </Grid.RowDefinitions>
+        <RichTextBox x:Name="headerText" 
+                     Background="Transparent"
+                     BorderThickness="0"
+                     IsReadOnly="True"
+                     IsDocumentEnabled="True"
+                     FontSize="12"
+                     FontWeight="Bold"
+                     VerticalAlignment="Top" 
+                     HorizontalAlignment="Left" 
+                     Margin="5,7,0,0" Width="382">
+            <FlowDocument>
+                <Paragraph>
+                    PowerPointLabs Version
+                    <Run x:Name="versionRunText">4.0.0.0</Run>
+                </Paragraph>
+            </FlowDocument>
+        </RichTextBox>
+        <Separator HorizontalAlignment="Left" Height="2" Margin="5,30,0,0" VerticalAlignment="Top" Width="382"/>
+        <RichTextBox x:Name="contentText" 
+                     Background="Transparent"
+                     BorderThickness="0"
+                     IsReadOnly="True"
+                     IsDocumentEnabled="True"
+                     FontSize="12"
+                     VerticalAlignment="Top" 
+                     HorizontalAlignment="Left" 
+                     Margin="5,37,0,0" Width="382">
+            <FlowDocument>
+                <Paragraph>
+                    Release date:
+                    <Run x:Name="releaseDateRunText">16 Jun 2017</Run>
```

Comment:
Ok i have set it to 31 Jan 1980 to be obvious that it is a placeholder.
Version number will be 9.9.9.9 too.